src/parse.c

```diff
@@ -667,6 +667,13 @@ parse_renderer(yaml_node_t* node, NetplanBackend* backend, GError** error)
 static gboolean
 handle_netdef_renderer(yaml_document_t* doc, yaml_node_t* node, const void* _, GError** error)
 {
+    if (cur_netdef->type == NETPLAN_DEF_TYPE_VLAN) {
+        if (strcmp(scalar(node), "sriov") == 0) {
+            cur_netdef->vf_filter = TRUE;
```

Comment:
+1 on the rename, I think it makes more sense.

As for removing vf_filter - I don't think that's possible. The main purpose of the vf_filter variable (which I guess I'll rename to sriov_vlan_filter maybe!) is to mark the given vlan definition to be handled directly via the netlink layer. So for SR-IOV VF devices it's possible to have both regular VLANs and the so-called hardware filtered VLANs. A VF device, as any device, can have any number of vlans, but can also have a special hardware one set if the user uses the `renderer: sriov` tag it its definition. Those can only be one-per-VF.

So even though that would look better, it would mean that VFs could then not have regular vlans, which we do not want. At least if I understand your idea correctly!

Reply:
Oh, I see. I wasn't aware of the possibility to have HW and SW defined VLANs on those devices. So let's keep it like this.